HTML/HTML.sublime-syntax

```diff
@@ -13,6 +13,7 @@ first_line_match: (?i)<(!DOCTYPE\s*)?html
 scope: text.html.basic
 
 variables:
+  attribute_name: (?:[^ "'>/=]+)
```

Comment:
The HTML5 validators I've tried:

- https://validator.w3.org/nu/#textarea
- https://html5.validator.nu/

seem to suggest that we should add `\t` to this because:

```html
<!DOCTYPE html>
<html>
<head>
<title>Test</title>
</head>
<body>
<p	class	id	=	hi i	d	=	"ok"></p>
<!-- all the whitespace inside the above `p` tag are tab characters -->
</body>
</html>
```

results in:

```
Error: Attribute i not allowed on element p at this point.
Error: Attribute d not allowed on element p at this point.
```

meaning that it doesn't see an attribute called `class\tid`, it sees `class` with no value and `id` with value `hi`, so scoping the tab character as `entity.other.attribute-name.html` seems wrong. Related to this, it should pop [when an unquoted value is encountered](https://github.com/sublimehq/Packages/blob/b85a691be63ca8daf4873a404f7d16d255fe5613/HTML/HTML.sublime-syntax#L308-L309), otherwise it incorrectly marks the rest of the tag contents as an unquoted string or at least until it reaches a quoted string.

Reply:
You're right — the tab, line feed, and form feed characters are listed in the [parsing algorithm](https://html.spec.whatwg.org/multipage/parsing.html#before-attribute-name-state). No idea why they're omitted in the other part of the spec.